### PR TITLE
Document godep updates better

### PR DESCRIPTION
**Pull request by thockin**
`godep update` doesn't work.  It just says 'nothing to update'.  Drop it, and
use The Big Hammer instead.

@quinton-hoole @lavalamp

### CODE REVIEW

File: docs/devel/development.md [hunk 1]
```diff
@@ -244,25 +244,46 @@ godep restore
...
+DEP=example.com/path/to/dependency
+godep get $DEP/...
 # Now change code in Kubernetes to use the dependency.
-hack/godep-save.sh
+./hack/godep-save.sh
+```
 
+To update an existing dependency is a bit more complicated.  Godep has an
+`update` command, but none of us can figure out how to actually make it work.
+Instead, this procedure seems work reliably:
```
Comment (quinton-hoole) [inline docs/devel/development.md]: typo: seems to work

**Comment by quinton-hoole** [comment]
LGTM barring minor typo.

**Comment by thockin** [comment]
Typo fixed.

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**